Italian/scappare/scopri-oggetto/esamina-stanza/esamina-stanza.md

```diff
@@ -4,3 +4,5 @@ Vicino al letto noti una scrivania di legno con alcuni fogli sparsi su di essa.
 [Esamini i fogli sulla scrivania](esamina-fogli/esamina-fogli.md)
 
 [Osservi il resto della stanza](controlla-stanza/controlla-stanza.md)
+
+[Esamini le pareti in cerca di passaggi] (./passaggi_stanza/passaggi-stanza.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.